app/Http/Controllers/PrintController.php

```diff
@@ -14,27 +14,23 @@ public function index() {
     }
 
     public function modify_balance(Request $request) {
```

Comment:
Just a comment: if we are changing slowly to Laravel's naming convention, could you change the name of the function to modifyBalance?

Reply:
I thought you decided to do codestyle changes only after the initial release.
https://github.com/luksan47/mars/issues/37#issuecomment-539968564

I think all code should be refactored in a single PR to follow the decided coding style.
If it were differently, the style would be inconsistent even in the same file, which is worse than the proposed solution - I think.

Reply:
Maybe @vendi95 could tell his opinion.

Reply:
On a second look, it should not be changed now, you are right.